util/fastCross3.m

```diff
@@ -0,0 +1,7 @@
+function X = fastCross3(U, V)
```

Comment:
In the process of moving the test case over from DRc, I discovered that at least for 2014a, cross() and fastCross3() have very similar performance. I'm going to get rid of this, since a very small performance boost during footstep planning probably doesn't warrant maintaining a whole new function.